kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/admin/UserTaskAdminServiceIntegrationTest.java

```diff
@@ -283,6 +284,44 @@ public void testReassignNotStartedWithBadTimeFormat() throws Exception {
         }
     }
 
+    @Test
+    public void testReassignNotStartedWithEmptyTimeFormat() throws Exception {
+        changeUser(USER_ADMINISTRATOR);
+        Map<String, Object> parameters = new HashMap<>();
+
+        Long processInstanceId = null;
+        try {
+            processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_EVALUATION, parameters);
+            List<TaskSummary> tasks = taskClient.findTasksAssignedAsBusinessAdministrator(USER_ADMINISTRATOR, 0, 10);
+            TaskSummary task = tasks.get(0);
+            OrgEntities reassign = OrgEntities.builder().users(Arrays.asList(USER_JOHN)).build();
+            assertExceptionContainsCorrectMessage(() -> userTaskAdminClient.reassignWhenNotStarted(CONTAINER_ID, task.getId(), "", reassign), "Error code: 400, message: \"Invalid time expression\"", "Invalid time expression");
```

Comment:
Please update the test to use error assertion defined here [1] to have such functionality in one place.

[1] https://github.com/kiegroup/droolsjbpm-integration/blob/a892e919650d52a62d93cbbb28fe0d43a3799622/kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/basetests/RestJmsSharedBaseIntegrationTest.java#L176-L191